crates/core_simd/tests/pointers.rs

```diff
@@ -1,4 +1,4 @@
-#![feature(portable_simd, strict_provenance)]
+#![feature(portable_simd, strict_provenance, exposed_provenance)]
```

Comment:
Please split this into a separate commit and in the commit message link to the PR where exposed_provenance was split out.

Reply:
@bjorn3 this should allow you to drop [the patch](https://github.com/rust-lang/rust/commit/1095e0165c36bd65f5fb3af9955e565137238b65) once this is merged and subtree synced.

Reply:
I can actually drop it as soon as this PR is merged. cg_clif's test suite currently directly clones this PR to test it rather than using the copy used by the standard library.